rasa_nlu/featurizers/regex_featurizer.py

```diff
@@ -87,6 +103,36 @@ def features_for_patterns(self, message):
         found = [1.0 if m is not None else 0.0 for m in matches]
         return np.array(found)
 
+    def _generate_lookup_regex(self, lookup_table):
+        """creates a regex out of the contents of a lookup table file"""
+        lookup_elements = lookup_table['elements']
+        elements_to_regex = []
+
+        # if it's a list, it should be the elements directly
+        if isinstance(lookup_elements, list):
+            elements_to_regex = lookup_elements
+
+        # otherwise it's a file path.
+        else:
+
+            try:
+                f = io.open(lookup_elements, 'r')
+            except IOError:
+                raise ValueError("Could not load lookup table {}".format(lookup_elements))
```

Comment:
could you change this to "Could not load lookup table {}. Make sure you've provided the correct path"